source/reference/limitations.txt

```diff
@@ -34,9 +34,10 @@ General Limitations
   are properly configured.
 - Other clients must not write to the destination cluster while
   ``mongosync`` is running.
-- If write blocking is disabled, the client must :ref:`prevent writes
-  <c2c-api-start>` to the source cluster before starting the commit
-  process.
+- If you want to start the :ref:`commit <c2c-api-commit>`
+  process and if you previously disabled :ref:`write blocking
```

Comment:
[nit]

I think we can remove the "if" on line 38.

Reply:
Done